packages/lit-element/package.json

```diff
@@ -45,34 +45,28 @@
     "@babel/plugin-proposal-decorators": "^7.2.3",
     "@babel/plugin-transform-typescript": "^7.2.0",
     "@types/chai": "^4.0.1",
-    "@types/mocha": "^5.2.4",
+    "@esm-bundle/chai": "^4.1.5",
+    "@types/mocha": "^8.0.3",
+    "@web/test-runner": "^0.7.11",
+    "mocha": "^8.1.1",
+    "prettier": "^2.0.5",
+    "rollup": "^2.26.4",
+    "rollup-plugin-filesize": "^9.0.2",
+    "rollup-plugin-terser": "^7.0.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "terser": "^5.2.1",
+    "typescript": "^4.0.2",
     "@webcomponents/shadycss": "^1.8.0",
     "@webcomponents/webcomponentsjs": "^2.2.3",
     "chai": "^4.0.2",
     "clang-format": "^1.2.4",
     "downlevel-dts": "^0.6.0",
     "lit-element-benchmarks": "^0.1.0",
-    "mocha": "^6.2.2",
     "np": "^5.2.1",
-    "rollup": "^1.26.0",
-    "rollup-plugin-filesize": "^6.2.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-terser": "^5.1.0",
     "tachometer": "^0.4.16",
     "tslint": "^5.20.1",
     "typedoc": "^0.17.0-3",
-    "typescript": "~3.8.2",
-    "uglify-es": "^3.3.9",
-    "wct-mocha": "^1.0.0",
-    "web-component-tester": "^6.9.2"
-  },
-  "typings": "lit-element.d.ts",
-  "typesVersions": {
-    "<3.8": {
-      "*": [
-        "ts3.4/*"
-      ]
-    }
+    "uglify-es": "^3.3.9"
```

Comment:
Can you remove this now? terser should be taking it's place

Reply:
Done.